tests/test_data_management.py

```diff
@@ -92,20 +92,30 @@ def test_build_method_with_new_drivers(self):
 
     def test_platform_serialization_from_imported_dict(self):
         """Test platform serialization by building a platform, saving it and then load it back again twice. Starting from a given dict."""
-        original_platform = ql.build_platform(Galadriel.runcard)
+        original_dict = Galadriel.runcard
```

Comment:
Just in case, can you do a `copy.deepcopy` of this dictionary? To make sure the changes below don't affect other tests.

Reply:
Agree, I'll do that!